plugins/s3-viewer-backend/src/service/S3Api.ts

```diff
@@ -167,7 +171,20 @@ export class S3Client implements S3Api {
     key: string,
   ): Promise<Readable> {
     const s3Client = this.getS3Client(endpoint, bucket);
-    return s3Client.getObject({ Bucket: bucket, Key: key }).createReadStream();
+    const { Body } = await s3Client.send(
```

Comment:
This upper-case assignment is a bit confusing, let's make it lower-case like `let { Body: body } = ...`.